include/glfw/window/window.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * -- GLFW thread safety key points --
+ *
+ * Initialization, termination, event processing and the creation and destruction of windows, cursors
+ * and OpenGL and OpenGL ES contexts are all restricted to the main thread.
+ *
+ * Because event processing must be performed on the main thread, all callbacks except for the error callback
+ * will only be called on that thread.
+ *
+ * All Vulkan related functions may be called from any thread.
+ *
+ */
+
+#pragma once
+
+#include <GLFW/glfw3.h>
```

Comment:
You should create a dedicated file for GLFW inclusion in common/ directory.
Additionly, as we are using GLFW with Vulkan exclusively there needs to be a macro definition before the header inclusion:
```cpp
#define GLFW_INCLUDE_VULKAN
#include <GLFW/glfw3.h>
```

Reply:
Actually we don't need **GLFW_INCLUDE_VULKAN**, because it just includes **vukan.h**.  
As we made **vulkan_include.h**, we need **glfw_include.h** to unify API headers we use.

Reply:
My mistake.